.ci/settings/settings.py

```diff
@@ -1,4 +1,3 @@
 CONTENT_ORIGIN = "http://localhost:8080/"
 ALLOWED_EXPORT_PATHS = ["/tmp"]
-ORPHAN_PROTECTION_TIME = 0
```

Comment:
What led us to this removal?

Reply:
Is the settings file even used?

Reply:
It's not needed. And running more with default settings is better, IMHO.